conf/mysql

```diff
@@ -0,0 +1 @@
+MYSQL_ROOT_PASSWORD=yourpassword
```

Comment:
I'm not a guy who knows a guy guy, but on topic, I shouldn't add a default file here because I know that people will not think about this a second and run with a default password